cmd/podman/machine/reset.go

```diff
@@ -3,18 +3,12 @@
 package machine
 
 import (
-	"bufio"
-	"fmt"
-	"os"
-	"strings"
-
 	"github.com/containers/common/pkg/completion"
 	"github.com/containers/podman/v5/cmd/podman/registry"
 	"github.com/containers/podman/v5/cmd/podman/validate"
 	"github.com/containers/podman/v5/pkg/machine"
-	"github.com/containers/podman/v5/pkg/machine/env"
+	provider2 "github.com/containers/podman/v5/pkg/machine/provider"
```

Comment:
Why is this renamed?

Reply:
Copy and paste from other file, I can change it.

Reply:
Turns out provider is already declared in machine.go, so this has to remain provider2